test/Interpreter/SDK/archiving_generic_swift_class.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-run-simple-swift | %FileCheck %s
+// RUN: %target-run-simple-swift -swift-version 3 | %FileCheck %s
```

Comment:
This isn't actually using Swift 3, which is why you had to update the command line parsing.